### PR TITLE
fixes for cassie

### PR DESCRIPTION
**Pull request by sprsquish**
Exclude old versions of finagle and util from cassie's dependencies.
Switch endTs to be where the current setup expects it to be.

### CODE REVIEW

**Comment by bmdhacks** [comment]
shipit

File: zipkin-common/src/main/scala/com/twitter/zipkin/storage/SpanStore.scala [hunk 1]
```diff
@@ -160,7 +160,7 @@ class InMemorySpanStore extends SpanStore {
         spans
     }) filter { span =>
       span.lastAnnotation match {
-        case Some(ann) => ann.timestamp <= endTs
+        case Some(ann) => ann.timestamp >= endTs
```
Comment (franklinhu) [inline zipkin-common/src/main/scala/com/twitter/zipkin/storage/SpanStore.scala:163]: What's the reasoning behind this change? The endTs argument is supposed to be an upper bound for "get me all traces before this time" not after.

Comment (sprsquish) [inline zipkin-common/src/main/scala/com/twitter/zipkin/storage/SpanStore.scala:163]: It's wrong in the current implementation. The current implementation uses it as the start of the slice [1][2]. I intend to fix it after moving to the new CassieSpanStore.

1) https://github.com/twitter/zipkin/blob/master/zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraIndex.scala#L147
2) https://github.com/twitter/cassie/blob/master/cassie-core/src/main/scala/com/twitter/cassie/ColumnFamily.scala#L148

Comment (sprsquish) [inline zipkin-common/src/main/scala/com/twitter/zipkin/storage/SpanStore.scala:163]: @franklinhu am I missing something somewhere though?

**Pull request closed without merging**